### PR TITLE
Modified mongodb.json to resolve "Not enough disk space" error

### LINKED ISSUE #9634: "Not enough disk space"
Challenge [Store Data in MongoDB](https://www.freecodecamp.com/challenges/store-data-in-mongodb) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

If you follow the tutorial directly, in the step called "Connect" it will ask you to start a server  using the command "mongod --port 27017 --dbpath=./data". Doing that will get you this error: http://i.imgur.com/cGy0iRP.png saying there isn't enough disk space avaiable. For this operation to work, you must attach "--smallfiles" to the mongod command.

### PR DESCRIPTION
**Pull request by Grantlyk**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Git-Squash) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #9634  
#### Description

I Modified the text in mongodb.json file to offer more support for students on c9.io and help them avoid a "Not enough disk space" error -- the language used was suggested by @BKinahan 

A follow up to this, an issue will need to be made to create the `descriptionEs` translation of these changes

### CODE REVIEW

**Comment by Bouncey** [comment]
### LGTM :+1: 

Can you please raise an issue for the Spanish translation and reference this PR with `#9647` in the issue description

Thanks for contributing :tada:

**Comment by Grantlyk** [comment]
@Bouncey Done 👍

**Comment by Bouncey** [comment]
:+1:  Thanks @Grantlyk

File: seed/challenges/03-back-end-development-certification/mongodb.json [hunk 1]
```diff
@@ -20,12 +20,12 @@
...
-        "Whenever you run a command that includes <code>mongod</code> on c9.io, use <code>./mongod</code> instead.",
+        "Whenever you run a command that includes <code>mongod</code> on c9.io, use <code>./mongod</code> instead. Together with the commands used in Step 11, this adds additional parameters to the <code>mongod</code> command to allow MongoDB to run in the C9 environment.",
         "Note that you can resize the c9.io's windows by dragging their borders.",
         "Make sure that you are always in your project's \"workspace\" directory. You can always navigate back to this directory by running this command: <code>cd ~/workspace</code>.",
         "You can view this Node School module's source code on GitHub at <a href='https://github.com/evanlucas/learnyoumongo'>https://github.com/evanlucas/learnyoumongo</a>.",
         "Complete \"Mongod\"",
-        "Complete \"Connect\"",
+        "Complete \"Connect\" <strong>Note:</strong> Use the command <code>mongod --port 27017</code> in place of <code>mongod --port 27017 --dbpath=./data</code> to avoid a duplicate parameter error. <code>--dbpath=./data</code> is already included in the <code>./mongod</code> command.",
```
Comment (raisedadead) [inline seed/challenges/03-back-end-development-certification/mongodb.json]: add a couple of breaks and take this into a new line

image

**Comment by raisedadead** [comment]
@Grantlyk thanks a lot for the fix, would you mind adding a couple of breaks in the code, currently the note looks little packed.

**Comment by bugron** [comment]
@Grantlyk any progress on this PR?

**Comment by raisedadead** [comment]
@bugron I am merging this and will add the `<br>` myself in a separate pull request

**Pull request merged**